test/ectest.c

```diff
@@ -1396,7 +1344,7 @@ static int nistp_single_test(int idx)
         || !TEST_false(EC_GROUP_have_precompute_mult(NISTP))
         /* now repeat all tests with precomputation */
         || !TEST_true(EC_GROUP_precompute_mult(NISTP, ctx))
-        || !TEST_true(EC_GROUP_have_precompute_mult(NISTP))
+        || !TEST_int_ge(EC_GROUP_have_precompute_mult(NISTP), 0)
```

Comment:
We can effectively strip this, right?
We are testing that the value is 0 or greater for a function that should return either 0 or 1.

Reply:
done in ab7aa90af9c1ceb4c8df71d33c4b4e5805efd6e2